File: check-active-template.js
const { PrismaClient } = require('@prisma/client');

const prisma = new PrismaClient();

async function main() {
  try {
    console.log('Fetching active handover template...\n');
    
    const activeTemplate = await prisma.templates.findFirst({
      where: {
        operation_type: 'handover',
        is_active: true,
      },
    });

    if (activeTemplate) {
      console.log('✓ Active template found:');
      console.log('  - File name:', activeTemplate.file_name);
      console.log('  - Storage provider:', activeTemplate.storage_provider);
      console.log('  - Storage bucket:', activeTemplate.storage_bucket);
      console.log('  - Storage path:', activeTemplate.storage_path);
      console.log('  - File size:', activeTemplate.file_size_bytes, 'bytes');
      console.log('  - Created at:', activeTemplate.created_at);
      console.log('  - Created by:', activeTemplate.created_by_user_id);
    } else {
      console.log('✗ No active handover template found in database!');
    }

    console.log('\nAll handover templates (including inactive):\n');
    const allTemplates = await prisma.templates.findMany({
      where: {
        operation_type: 'handover',
      },
    });

    allTemplates.forEach(t => {
      console.log(`  ${t.is_active ? '✓ ACTIVE' : '○ INACTIVE'}: ${t.file_name}`);
      console.log(`    UUID: ${t.uuid}`);
      console.log(`    Path: ${t.storage_path}`);
      console.log(`    Size: ${t.file_size_bytes} bytes`);
      console.log('');
    });

  } catch (error) {
    console.error('Error:', error);
  } finally {
    await prisma.$disconnect();
  }
}

main();

File: check_active_template.py
import os
os.environ['PYTHONPATH'] = '/d/next-postgres-starter'

import sys
sys.path.insert(0, '/d/next-postgres-starter')

# Use Supabase client directly
from supabase import create_client

url = os.environ.get('NEXT_PUBLIC_SUPABASE_URL')
key = os.environ.get('SUPABASE_SERVICE_ROLE_KEY')

if not url or not key:
    print(f"Missing env vars. URL: {'OK' if url else 'MISSING'}, KEY: {'OK' if key else 'MISSING'}")
    sys.exit(1)

client = create_client(url, key)

# Query active template
result = client.table('templates').select('*').eq('operation_type', 'handover').eq('is_active', True).execute()

print("Active handover template:")
for template in result.data:
    print(f"  file_name: {template['file_name']}")
    print(f"  storage_path: {template['storage_path']}")
    print(f"  storage_provider: {template['storage_provider']}")
    print(f"  file_size_bytes: {template['file_size_bytes']}")
    print(f"  created_at: {template['created_at']}")
